tests/ops/fixtures/saas/zendesk_fixtures.py

```diff
@@ -27,6 +28,8 @@ def zendesk_secrets(saas_config):
         "domain": pydash.get(saas_config, "zendesk.domain") or secrets["domain"],
         "username": pydash.get(saas_config, "zendesk.username") or secrets["username"],
         "api_key": pydash.get(saas_config, "zendesk.api_key") or secrets["api_key"],
+        "page_size": pydash.get(saas_config, "zendesk.page_size")
```

Comment:
This should have been caught before but it actually throws an error now if this value is missing, which is the correct behavior.